### PR TITLE
Clean the f1 graph dataset and add database visualize function

### PR DESCRIPTION
**Pull request by zechengz**
* Drop `constructor_results` status columns as > ~99.9% are nulls
* Add circuits alt column as this may be useful
* Convert drivers date of birth to timestamp data type in the pandas dataframe
* Add database schema visualization utils function

**rel-f1-position:**

| Name      | Dataset     | R2       | MAE        | RMSE       |
|-----------|-------------|----------|------------|------------|
| Updated    | Validation  | 0.267991 | 3.179950   | 3.966508   |
| Updated    | Test        | 0.143451 | 4.015781   | 4.822209   |
| Original  | Validation  | 0.266184 | 3.202419   | 3.971403   |
| Original  | Test        | 0.073817 | 4.108763   | 5.014392   |

**rel-f1-dnf:**

| Name      | Dataset     | Average Precision | Accuracy   | F1          | ROC AUC     |
|-----------|-------------|-------------------|------------|-------------|-------------|
| Updated    | Validation  | 0.436570          | 0.765018   | 0.452675    | 0.725098    |
| Updated    | Test        | 0.465119          | 0.702279   | 0.515081    | 0.719187    |
| Original  | Validation  | 0.418985          | 0.803887   | 0.379888    | 0.725098    |
| Original  | Test        | 0.461981          | 0.722222   | 0.306050    | 0.731884    |

**rel-f1-qualifying:**

| Name      | Dataset     | Average Precision | Accuracy   | F1          | ROC AUC     |
|-----------|-------------|-------------------|------------|-------------|-------------|
| Updated   | Validation  | 0.432659          | 0.768707   | 0.464567    | 0.760567    |
| Updated   | Test        | 0.362411          | 0.804408   | 0.377193    | 0.796254    |
| Original  | Validation  | 0.546746          | 0.811224   | 0.554217    | 0.795954    |
| Original  | Test        | 0.350454          | 0.804408   | 0.377193    | 0.773509    |

### CODE REVIEW

File: examples/gnn_node.py [hunk 1]
```diff
@@ -47,6 +47,9 @@
 dataset: RelBenchDataset = get_dataset(name=args.dataset, process=True)
 task: NodeTask = dataset.get_task(args.task, process=True)
 
+# Visualize the db
+dataset.db.visualize(f"{args.dataset}.png")
```
Comment (weihua916) [inline examples/gnn_node.py:51]: Not sure if we need this in example code.

**Review by weihua916 — COMMENTED** [review]
It'd be nice to double-check if this dataset change would change our experimental results.
cc @rishabh-ranjan 

What's the motivation to remove a bunch of columns in the first place? Was it for model performance improvement?

File: examples/gnn_node.py (hunk 1 above)
Comment (zechengz) [reply examples/gnn_node.py:51]: Will remove

**Comment by zechengz** [comment]
@weihua916 , thanks for the comment.
Previously I removed some unique value columns (such as `driverRef` name) from the drivers (entity) table as they are unique for each entity. But it seems that they are quite useful (after removing them model performance dropped) I think they may serve as something like shallow embeddings.
So I converted some changes back and reran the experiments (updated in the description above) and listed all changes in the description. As the dataset is quite small, the experiments results are not very stable. But IMO they are quite similar.

**Review by weihua916 — APPROVED** [review]

**Pull request merged**